docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
 
 <img src="../img/side-bar/3D-mode.jpg" class="ms-docimage"/>
 
 ## Identify Tool
 
-The Identify tool  <img src="../img/button/identify.jpg" class="ms-docbutton"/> allows you to interact with the map by clicking on features to get information stored in the attribute table of the queried feature. It is active by default as the green color suggests.
-
-* **Add** a layer to the map (e.g. USA Population).
+Getting information about the layers, which have been loaded into the map, is possible thanks to the **Identify** button <img src="../img/button/identify.jpg" class="ms-docbutton"/>, through which the attribute table, where the characteristics of the layer are stored, is queried. 
+The tool is active by default, as suggested by the green of the button. Therefore by clicking on a point on the level opens a modal window on the right side. On the window are dispalyed the requested information and the coordinates of the point.
 
-* **Click** on a feature of the layer added previously. A pop-up page will open on the right showing the requested information and the coordinates of the point.
+<img src="../img/side-bar/identify-tool.jpg" class="ms-docimage"/>
 
-<img src="../img/side-bar/get_feature_info.jpg" class="ms-docimage"/>
+The returned information are viewed in Text plain format by default. It is possible to change the format selecting the <img src="../img/button/setting_button.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu) where the user can select, from the list menu *Identify response format*, three different formats: **TEXT**, **HTML** and **PROPERTIES**.
 
-The returned information are viewed in Text plain format by default. In order to change it:
+<img src="../img/side-bar/format-options.jpg" class="ms-docimage" style="max-width:400px;"/>
 
-* **Navigate** into *Settings* from the main menu bar.
+The information will be returned in the format chosen by the user. For exaple with *PROPERTIES* format as follows:
 
-<img src="../img/side-bar/settings.jpg" class="ms-docimage"/>
-
-* **Select** from the list menu *Identify response format* another format (e.g. HTML) then **Click** on a feature.
-
-<img src="../img/side-bar/info_formats.jpg" class="ms-docimage" style="max-width:500px;"/>
-
-* The informations will be returned as HTML accordingly.
-
-<img src="../img/side-bar/html_info.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/format_example.jpg" class="ms-docimage"/>
 
 !!! warning
     This global settings could be overwritten by a layer-specific configuration (see [Feature Info Form](layer-settings.md#feature-info-form)).
 
-Below the point coordinates there are three icons:
 
-* **More Info** <img src="../img/button/more_info_icon.jpg" class="ms-docbutton"/> shows a popup window displaying the point address.
+In addition to the features, the following information is provided by the *Identify Tool*: 
```

Comment:
In addition to the layers information, the following are provided by the *Identify Tool*: